python/cuml/cuml/manifold/umap/umap.pyx

```diff
@@ -522,7 +522,8 @@ class UMAP(Base, InteropMixin, CMajorInputTagMixin, SparseInputTagMixin):
         sparse array (preferably CSR/COO). This feature allows
         the precomputation of the KNN outside of UMAP
         and also allows the use of a custom distance function. This function
-        should match the metric used to train the UMAP embeedings.
+        should match the metric used to train the UMAP embeedings. For efficient
+        memory usage, the precomputed knn graph should be given as CPU arrays.
```

Comment:
1. I think the term `CPU arrays` is actually ill-defined and could lead to confusion. I'd say "CPU-accessible arrays such as numpy arrays" might be better.
2. I would adopt terminology "For most efficient memory usage" rather than "For efficient memory usage" to not imply that our memory usage is generally inefficient otherwise.

Reply:
Done!